### ISSUE TITLE
Modificare development_status dei moduli di fatturazione elettronica

### ISSUE THREAD

**Issue by OpenCode** [OCA/l10n-italy#2816]
Modificare la maturità dei moduli di fatturazione elettronica `development_status` da `Beta` a `Stable`

* [ ] 12 - [l10n_it_fatturapa](https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_fatturapa)
* [ ] 14 - [l10n_it_fatturapa](https://github.com/OCA/l10n-italy/tree/14.0/l10n_it_fatturapa)
...

**Comment by tafaRU** [comment]
Quando decideremo di farlo dovremo tenere conto anche del fatto che in base a:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst#3stable-modules

ed in particolare a:

> Must depend only on Stable and Mature modules..

per far si che il seguente check venga superato:

https://github.com/OCA/l10n-italy/blob/5f20297cf877375d5f5af5d93329ed309eb024d2/.github/workflows/test.yml#L61

```
      - name: Check development status
        run: manifestoo -d . check-dev-status --default-dev-status=Beta
```

si renderà verosimilmente necessario dover modificare il livello di maturità anche delle sue dipendenze oppure attendere la risoluzione di https://github.com/OCA/oca-addons-repo-template/issues/83

**Comment by OpenCode** [comment]
> Quando decideremo di farlo dovremo tenere conto anche del fatto che in base a:
> 
> https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst#3stable-modules
> 
> ed in particolare a:
> 
> > Must depend only on Stable and Mature modules..
> 
> per far si che il seguente check venga superato:
> 
> https://github.com/OCA/l10n-italy/blob/5f20297cf877375d5f5af5d93329ed309eb024d2/.github/workflows/test.yml#L61
> 
> si renderà verosimilmente necessario dover modificare il livello di maturità anche delle sue dipendenze oppure attendere la risoluzione di [OCA/oca-addons-repo-template#83](https://github.com/OCA/oca-addons-repo-template/issues/83)

Certo. Aggiungo che parliamo di moduli in produzione presso tantissime installazioni e che sia verosimile che possano essere considerati stabili ormai.

**Comment by francesco-ooops** [comment]
@OpenCode se vuoi fare una PR https://github.com/OCA/oca-addons-repo-template/issues/83 è stato risolto
